rest-api/api_util.py

```diff
@@ -108,6 +114,30 @@ def format_json_enum(obj, field_name):
   else:
     obj[field_name] = UNSET
 
+def get_site_id_from_google_group(self, obj):
```

Comment:
As mentioned previously: you probably don't want to  pass in "self" as a first parameter here, as that's used normally when dealing with member methods on classes.  Just pass in the DAO object after obj, like in format_json_hpo. 

Also, I think rather than modifying the passed in object, it would be better in these cases to return it (or None). The only reason we had format_<STUFF> above modify the object in place is because we often create a dictionary first from a SQLAlchemy object and format certain fields in it before returning it... when converting from resource JSON, it makes more sense to construct a new SQLAlchemy object, and leave the original JSON alone. (Particularly as sometimes we save that JSON in the DB separate from the other stuff.)